cli/injector/main.go

```diff
@@ -809,33 +863,30 @@ func getPodResourceVersion() (string, error) {
 // if it has changed ID, the container just restarted and need to be reinjected
 func updateTargetContainersAndDetectChange(pod *v1.Pod) (bool, error) {
 	var err error
-
 	// transform map of targetContainer info (name, id) to only an array of names
 	targetContainerNames := []string{}
 	for name := range disruptionArgs.TargetContainers {
 		targetContainerNames = append(targetContainerNames, name)
 	}
 
-	// update map of targetContainer info (name, id)
-	disruptionArgs.TargetContainers, err = utils.GetTargetedContainersInfo(pod, targetContainerNames)
-	if err != nil {
-		log.Warnw("couldn't get containers info. Waiting for next change to reinject", "err", err)
+	log.Infow("disruption args containers BEFORE updating them", "target_containers", disruptionArgs.TargetContainers)
 
-		return false, err
+	if disruptionArgs.TargetContainers, err = v1beta1.TargetedContainers(pod, targetContainerNames); err != nil {
+		return false, fmt.Errorf("unable to get targeted containers info. Waiting for next change to reinject: %w", err)
 	}
 
-	// Determine if reinjection is needed
-	for ctnName, ctnID := range disruptionArgs.TargetContainers {
-		// we don't check for init containers
-		if ctnName == chaosInitContName && disruptionArgs.OnInit {
-			continue
+	log.Infow("disruption args containers AFTER updating them", "target_containers", disruptionArgs.TargetContainers)
```

Comment:
Do lines 872 and 878 need to be at Info? It feels like we could tolerate at least one at debug

Reply:
Indeed.
I'm also wondering why do we send Debug level to Datadog?
Do we have specific log pipeline to purge them more quickly?

Reply:
umm, at least for me, it's strictly for filtering? I always have the logs from all clusters tailed, and it's much nicer to use INFO to skim across everywhere, and use DEBUG for when inspecting an individual disruption

Reply:
👍